src/main.ts

```diff
@@ -246,6 +246,10 @@ export interface MynahUIProps {
     tabId: string,
     buttonId: string,
     eventId?: string) => void;
+  upDownArrowKeyPress?: (
+    tabId: string,
+    direction: 'up' | 'down',
+    eventId?: string) => void;
```

Comment:
huh? what is `string)`, isn't there a parenthesis missing? (same for line 248)

Reply:
where is the open? :)

Reply:
Not sure I got this. Maybe its just a formatting on github review? 
Here's what I see:
```
onFileActionClick?: (
    tabId: string,
    messageId: string,
    filePath: string,
    actionName: string,
    eventId?: string) => void;
 onTabBarButtonClick?: (
    tabId: string,
    buttonId: string,
    eventId?: string) => void;
 onUpDownArrowKeyPress?: (
    tabId: string,
    direction: 'up' | 'down',
    eventId?: string) => void;
```